gitbutler-ui/src/lib/backend/aiService.test.ts

```diff
@@ -0,0 +1,223 @@
+import { AnthropicAIClient } from './aiClients/anthropic';
+import { ButlerAIClient } from './aiClients/butler';
+import { OpenAIClient } from './aiClients/openAI';
+import { AIService, AnthropicModelName, KeyOption, ModelKind, OpenAIModelName } from './aiService';
+import { getCloudApiClient } from './cloud';
+import { expect, test, describe, vi } from 'vitest';
+import type { AIClient } from './aiClient';
+import type { GitConfigService } from './gitConfigService';
+
+const defaultGitConfig = Object.freeze({
+	'gitbutler.aiModelProvider': ModelKind.OpenAI,
+	'gitbutler.aiOpenAIKeyOption': KeyOption.ButlerAPI,
+	'gitbutler.aiAnthropicKeyOption': KeyOption.ButlerAPI,
+	'gitbutler.aiOpenAIKey': undefined,
+	'gitbutler.aiOpenAIModelName': OpenAIModelName.GPT35Turbo,
+	'gitbutler.aiAnthropicKey': undefined,
+	'gitbutler.aiAnthropicModelName': AnthropicModelName.Haiku
```

Comment:
Just realized we probably want to tie this to the new enum?

Reply:
Yes - I was going to do that 👍